### PR TITLE
Team page modified

### PR DESCRIPTION
**Pull request by karthiks2611**
For mobile users, navigating from team.html to index.html was not working, so added a fix for this

### CODE REVIEW

**Comment by karthiks2611** [comment]
BEFORE

Before



AFTER

After

**Comment by karthiks2611** [comment]
BEFORE

Before


AFTER
After

File: team.html [hunk 1]
```diff
@@ -37,6 +37,14 @@
         </a>
         <i class="fas fa-chevron-circle-down burger" id="nav-toggler"></i>
     </nav>
+    <div class="dropdown" id="dropdown">
+        <ul>
+            <li><a href="./index.html">Home</a></li>
+            <li class="active-dropdown">Our Team</li>
+            <li><a href="https://opensrc.cyscomvit.com/leaderboard">Leaderboard</a></li>
+            <li>Blogs</li>
```
Comment (kunatastic) [inline team.html:45]: please add https://medium.cyscomvit.com/ link for the blogs tab

**Review by kunatastic — APPROVED** [review]
LGTM

**Pull request merged**